### PR TITLE
Update protocol.txt

### PR DESCRIPTION
**Pull request by richlv**
fix minor typo

### CODE REVIEW

File: doc/protocol.txt [hunk 1]
```diff
@@ -711,7 +711,7 @@ other stats command.
 |-------------------+----------+----------------------------------------------|
 | Name              | Type     | Meaning                                      |
 |-------------------+----------+----------------------------------------------|
-| maxbytes          | size_t   | Maximum number of bytes allows in this cache |
+| maxbytes          | size_t   | Maximum number of bytes allowed in this cache |
```
Comment (greexon) [inline doc/protocol.txt]: Table doesn't have flat border at the right side.

Comment (richlv) [inline doc/protocol.txt]: oops, i'm very sorry about that... should be fixed now :)

**Comment by dormando** [comment]
in `next`. trivial typo fixes are committed by us and credited vs pulled

**Comment by dormando** [comment]
released in 1.4.34

**Pull request closed without merging**